db/migrate/20220602025217_add_is_parent_to_eligibilities.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsParentToEligibilities < ActiveRecord::Migration[6.1]
+  def change
+    add_column :eligibilities, :is_parent, :boolean, :default => false
```

Comment:
In theory we don't need this -- you could query to see if an eligibility has any children and if so consider it a parent -- and data denormalization is generally not great but I think it should be fine in this case.